tools/scripts/README.md

```diff
@@ -30,6 +30,11 @@ be registered in the determined repo. Here is the process:
    artifact's `old` value to the previous `new` value, and set the new `new`
    value to the artifact produced by CI.
```

Comment:
non-blocking: not part of this PR, but could help to call that this typically includes task containers and agent AMIs.

Reply:
done